modyn/supervisor/supervisor.py

```diff
@@ -359,7 +387,17 @@ def _run_training(self, trigger_id: int) -> None:
         if "evaluation" in self.pipeline_config:
             evaluations = self.grpc.start_evaluation(trained_model_id, self.pipeline_config)
             self.grpc.wait_for_evaluation_completion(self.current_training_id, evaluations)
-            self.grpc.store_evaluation_results(self.eval_directory, self.pipeline_id, trigger_id, evaluations)
+
+            writer_names: set[str] = set()
+            if "result_writers" in self.pipeline_config["evaluation"]:
+                writer_names.update(self.pipeline_config["evaluation"]["result_writers"])
+
+            writers: list[AbstractEvaluationResultWriter] = []
+            for writer_handler in self.supported_evaluation_result_writers:
+                if writer_handler.get_name() in writer_names:
```

Comment:
hmm, here we iterate over all handlers, check if we want them, and instantiate them. an alternative would be to iterate over all specified handlers, and instantiate them (e.g. with a string -> class map). WDYT?